### ISSUE TITLE
8.9 Release Tracker

### ISSUE THREAD

**Issue by Pessimistress** [visgl/deck.gl#7635]
## Remaining PRs/work items

- [x] TerrainExtension #7195 @Pessimistress
  + [x] Codepen example for documentation #7660 
- [x] CollideExtension #7374 @felixpalmer
  + [x] Codepen example for documentation #7660 
- [x] WMS support #7575 @ibgreen
- [x] loaders.gl 3.3 @belom88 
- [ ] probe.gl 4.0 @ibgreen 
  + [x] publish probe.gl 4.0
  + [ ] bump loaders.gl https://github.com/visgl/loaders.gl/pull/2371
  + [ ] Bump luma.gl https://github.com/visgl/luma.gl/pull/1729 https://github.com/visgl/luma.gl/pull/1730
  + [ ] Bump deck.gl https://github.com/visgl/deck.gl/pull/7686 
- [x] Fix website build @Pessimistress 
- [x] New website examples
  + [x] Mask, collide, terrain @alasarr 
  + [x] WMS @ibgreen 
- [x] API Audit - Feb 21, 2022
  + [x] CollisionFilterExtension #7675 
  + [x] WMSLayer #7677
  + [x] TerrainExtension #7678 

## Release checklist

Target release date: February 2023

- [x] Publish alpha/beta
- [ ] Bump all examples to latest alpha/beta
- [x] Verify whats-new
- [x] Verify upgrade-guide
- [x] Verify all issues in milestone are closed or move out
- [x] Verify all API changes are documented
- [x] Test website (docs and example) on [staging](https://pessimistress.github.io/deck.gl/)
    - [x] Testing on Chrome
    - [x] Testing on Safari
    - [x] Testing on Firefox
    - [x] Testing on Edge
    - [x] Testing on iOS
    - [x] Testing on Android
- [x] Cut 8.9-release branch after all fixes
- [x] Update website doc urls to point to `8.9-release` branch
- [x] Publish 8.9 prod version.

**Comment by ibgreen** [comment]
Added library upgrade checkmarks

**Comment by adrianes21** [comment]
Found a bug. While testing the new deck.gl version I tested the integration with CARTO and using dark mode it seems the labels in the layer are not visible until you click on them.


![](https://user-images.githubusercontent.com/8087799/220601651-7df5e9cc-8bb4-4f92-9fca-1b1035cc3366.png)

![](https://user-images.githubusercontent.com/8087799/220601716-2485ec25-5948-45ef-a7f2-09a46ede723d.png)

Expected Behavior
The labels in the legend should be visible in both modes

Steps to Reproduce
Go to https://pessimistress.github.io/deck.gl/examples/carto and activate dark mode. Label in legend are not there

Environment
Framework version: 8.9.0-beta. latest
Browser: Chrome, Safari, Firefox
OS:110.0.5481.100, 16.2, 109.0.1
